M2/Macaulay2/d/actors3.d

```diff
@@ -1062,7 +1062,7 @@ atan(e:Expr):Expr := (
      is x:RRicell do toExpr(atan(x.v))				    -- # typical value: atan, RRi, RRi
      is a:Sequence do if length(a) == 2 then buildErrorPacket("atan(x,y) has been replaced by atan2(y,x)")
```

Comment:
Pretty sure we can delete this line by now ... but no need to do it here and run the actions all over again.